pdr_backend/pdr_dashboard/callbacks/callbacks_home.py

```diff
@@ -225,3 +237,62 @@ def select_or_clear_all_predictoors(_, __, rows):
 
         ctx = dash.callback_context
         return select_or_clear_all_by_table(ctx, "predictoors_table", rows)
+
+    @app.callback(
+        [
+            Output("predictoor_config_modal", "is_open"),
+            Output("predictoor_addrs", "value"),
+        ],
+        [
+            Input("configure_predictoors", "n_clicks"),
+            Input("predictoor-addrs-local-store", "data"),
+        ],
+        prevent_initial_call=True,
+    )
+    def open_predictoors_config_modal(n_clicks, predictoor_addrs):
+        """
+        Select or clear all rows in the feeds table.
+        """
+        if len(predictoor_addrs) > 0:
+            predictoor_addrs = "\n".join(predictoor_addrs)
+        else:
+            predictoor_addrs = ""
+        if n_clicks <= 0:
+            return [dash.no_update, predictoor_addrs]
+
+        return [True, predictoor_addrs]
+
+    @app.callback(
+        Output("predictoor-addrs-local-store", "data"),
+        Input("save_predictoors", "n_clicks"),
+        State("predictoor_addrs", "value"),
+    )
+    def save_predictoors_to_browser_storage(n_clicks, value):
+        if not value and not n_clicks:
+            return dash.no_update
+        addresses = value.split("\n")
```

Comment:
I don't think we need to be passing strings, State can accept a list of strings AFAIK

Reply:
did you look into this? Can you have a list of strings instead of a customly created string?

Reply:
I don't get it, isn't the 'addresses' a list of string?

Reply:
I was talking about the `value`. But you are right, that's the value of the textarea. Marking this as resolved